src/utils/index.js

```diff
@@ -9,6 +9,7 @@ module.exports.checkHeadsetConnected = require('./checkHeadsetConnected');
 module.exports.debug = require('./debug');
 module.exports.entity = require('./entity');
 module.exports.forceCanvasResizeSafariMobile = require('./forceCanvasResizeSafariMobile');
+module.exports.functions = require('./functions');
```

Comment:
Actually might be more consistent to have this as `function` rather than `functions`. (I imagine we'd have `object` and we already have `entity`). Hopefully not too much hassle to replace.